Code/src/main/resources/catalog/catalog.xml

```diff
@@ -18,105 +35,584 @@
+        		longest prefix (possibly empty) of xs of element that do not satisfy p and second element is the remainder of the list.
+        	</function>
+        	<function name="notElem" signature="Eq a => a -> [a] -> Bool">
+        		notElem is the negation of elem.
+        	</function>
+        	<function name="lookup" signature="Eq a => a -> [(a, b)] -> Maybe b">
+        		lookup key assocs looks up a key in an association list.
+        	</function>
+        	<function name="zip" signature="[a] -> [b] -> [(a, b)]">
+        		zip takes two lists and returns a list of corresponding pairs. If one input list is short,
+        		excess elements of the longer list are discarded.
+        	</function>
+        	<function name="zip3" signature="[a] -> [b] -> [b] -> [(a, b, c)]">
+        		zip3 takes three lists and returns a list of triples, analogous to zip.
+        	</function>
+        	<function name="zipWith" signature="(a -> b -> c) -> [a] -> [b] -> [c]">
+        		zipWith generalises zip by zipping with the function given as the first argument, instead of a tupling function.
+        		For example, zipWith (+) is applied to two lists to produce the list of corresponding sums.
+        	</function>
+        	<function name="zipWith3" signature="(a -> b -> c -> d) [a] -> [b] -> [c] -> [d]">
+        		The zipWith3 function takes a function which combines three elements, as well as three
+        		lists and returns a list of their point-wise combination, analogous to zipWith.
+        	</function>
+        	<function name="unzip" signature="[(a, b)] -> ([a], [b])">
+        		unzip transforms a list of pairs into a list of first components and a list of second
+        		components.
+        	</function>
+        	<function name="unzip3" signature="[(a, b, c)] -> ([a], [b], [c])">
+        		the unzip3 function takes a list of triples and returns three lists, analogous to unzip.
+        	</function>
+        	<function name="lines" signature="String -> [String]">
+        		lines breaks a string up into a list of strings at newline characters.
+        		The resulting strings do not contain newlines.
+        	</function>
+        	<function name="words" signature="String -> [String]">
+        		words breaks a string up into a list of words, which were delimited by white space.
+        	</function>
+        	<function name="unlines" signature="[String] -> String">
+	        	unlines is an inverse operation to lines. It joins lines, after appending a
+	        	terminating newline to each.
+        	</function>
+        	<function name="unwords" signature="[String] -> String">
+        		unwords is an inverse operation to words. It joins words with separating spaces.
+        	</function>
+        </category>
+        <category name="Converting to and from String">
+        	<function name="showsPrec" signature="Show a => Int -> a -> ShowS">
```

Comment:
Apparently, ShowS is really `String -> String` - you could put that here directly, or drop the ShowS functions, your call.